src/commands/init/index.ts

```diff
@@ -47,26 +58,78 @@ export default class Init extends Command {
             required: false,
             default: false,
         }),
+        package: Flags.string({
+            char: 'p',
+            aliases: ['package'],
+            description: '',
+            required: false,
+            default: '',
+        }),
     };
 
     async run(): Promise<void> {
         const { flags } = await this.parse(Init);
         this.log(`Initializing Velocitas packages ...`);
-        const projectConfig = this.initializeOrReadProject();
+        const projectConfig = this._initializeOrReadProject();
 
         const appManifestData = AppManifest.read(projectConfig.getVariableMappings().get(APP_MANIFEST_PATH_VARIABLE));
 
-        await this.ensurePackagesAreDownloaded(projectConfig, flags.force, flags.verbose);
-        projectConfig.validateUsedComponents();
+        let packageConfigs = projectConfig.getPackages(false);
+        if (flags.package !== '') {
+            const requestedPackageConfig = this._parsePackageConfig(flags.package);
+            await this._resolveVersion(requestedPackageConfig, flags.verbose);
+            await this._modifyProjectConfig(requestedPackageConfig, projectConfig, flags.force, flags.verbose);
+            await this._ensurePackageIsDownloaded(requestedPackageConfig, flags.force, flags.verbose);
+            this._addComponentsToProjectConfig(requestedPackageConfig, projectConfig);
+            // projectConfig.validateUsedComponents();
```

Comment:
uncommented because installing a single package (if not starting with a blank .velocitas.json) will always result in errors, e.g.: 

packageA -> CompA
packageB -> CompB
packageC -> CompC

init -p packageB
=> Error: Component not found CompA (and CompC)
init -p packageA
=> Error: Component not found CompC
init -p packageC
=> OK

Maybe this is what we want - throw errors until everything is installed correctly, but I'm unsure here